agentops/client.py

```diff
@@ -264,11 +266,13 @@ def end_session(self,
 
         self._session.video = video
         self._session.end_session(end_state, end_state_reason)
-        token_cost = self._worker.end_session(self._session)
+        getcontext().prec = 6
+        token_cost = Decimal(self._worker.end_session(self._session))
```

Comment:
Please make sure you test this. 

Simply converting to decimal doesn't actually get rid of floating point precision from the floats. For example:

```
In [10]: print(Decimal(0.01))
0.01000000000000000020816681711721685132943093776702880859375

In [11]: print(Decimal(str(0.01)))
0.01
```

Casting the float to string before will solve this.